### PR TITLE
Feature/5 jbuilder and cors

### PR DESCRIPTION
**Pull request by lisset-orozco**
- [x] create seed

- [x] layouts with jbuilder

- [x] show errors

- [x] represent resources in json

- [x]  refactor error_handler

- [x] partials in jbuilder

- [x] refactor json views

- [x]  cors headers

### CODE REVIEW

File: app/controllers/api/v1/users_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class UsersController < ApiV1Controller
 
       def create
         # params = { auth: { provider: , uid: } }
-        return error_message unless params[:auth]
+        return render_message unless params[:auth]
 
         @user = User.from_omniauth(params.fetch(:auth))
```
Comment (joalbertg) [inline app/controllers/api/v1/users_controller.rb:13]: use memoization to user

File: app/controllers/api/v1/users_controller.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@ class UsersController < ApiV1Controller
...
         @token = TokenService.new(user: user).object
```
Comment (joalbertg) [inline app/controllers/api/v1/users_controller.rb]: use memoization to token or callbacks to create and show actions

File: app/controllers/api/v1/users_controller.rb [hunk 3]
```diff
@@ -19,7 +19,7 @@ def create
 
       def show
         # params = { auth: { provider: , uid: } }
-        return error_message unless params[:auth]
+        return render_message unless params[:auth]
 
         @user = User.from_omniauth(params.fetch(:auth))
```
Comment (joalbertg) [inline app/controllers/api/v1/users_controller.rb]: use memoization to user or callbacks to create and show actions

**Review by joalbertg — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/users_controller.rb (hunk 3 above)
Comment (lisset-orozco) [reply app/controllers/api/v1/users_controller.rb]: done

File: app/controllers/api/v1/users_controller.rb (hunk 1 above)
Comment (lisset-orozco) [reply app/controllers/api/v1/users_controller.rb:13]: done

File: app/controllers/api/v1/users_controller.rb (hunk 2 above)
Comment (lisset-orozco) [reply app/controllers/api/v1/users_controller.rb]: done

**Review by joalbertg — APPROVED** [review]

**Pull request merged**